### PR TITLE
Adds "Import configuration only" option

### PR DESCRIPTION
**Pull request by jsiebern**
### Description of the problem being solved:
When following a creators build guide you always come to the point where you import your own profile in order to compare to the creators guide PoB. Manually filling out the configuration can be quite tedious (you usually have to alt+tab between multiple windows). This PR attempts to make this easier by adding the `Import configuration only` option to the import dropdown. This way you can import your own build, save and then import the guides configuration on top.

### Steps taken to verify a working solution:
- Import a character
- Save
- Go to import again and enter a URL/Code, then select "Import configuration only" and click "Import"
- On success you will be taken to the configuration panel with the updated values

### Before screenshot:
image

### After screenshot:
image

### CODE REVIEW

**Comment by Regisle** [comment]
Could you not just import the guide first and then your character over the top?, as that deletes the items/skills/tree of the guide anyway and leaves you in the same endstate.

**Comment by jsiebern** [comment]
Good point! Yeah, you could definitely import the guide first and then your character. I've run into bugs doing that before, but it was a while ago. Personally, I find the 'Import Configuration Only' option a bit more user-friendly. If the consensus is that the existing approach works fine, I understand and you're welcome to close the PR.

File: src/Modules/Build.lua [hunk 1]
```diff
@@ -731,6 +731,29 @@ function buildMode:Init(dbFileName, buildName, buildXML, convertBuild)
...
+	end
+	for _, node in ipairs(dbXML[1]) do
+		-- Check if there is a saver that can load this section
+		if node.elem == "Config" then
+			local saver = self.savers[node.elem]
+				if saver:Load(node, self.dbFileName) then
+					self:CloseBuild()
+					return true
+				end
+		end
```
Comment (Wires77) [inline src/Modules/Build.lua:751]: Spacing seems off here.  Plus you can remove the comment, as it no longer applies here.

**Review by Wires77 — CHANGES_REQUESTED** [review]
I get this error when trying to import the config.  Still imports it, but shows the error.  One other thing you might want to change is have any previous configuration cleared out first, as this just adds to what is already there.
image

**Comment by jsiebern** [comment]
Closing this since there will be an update in the near futuer that adds support  for multiple configurations. I might revisit this feaure once that is implemented.

**Pull request closed without merging**